compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/loops/ProgressionHandlers.kt

```diff
@@ -247,7 +245,7 @@ internal abstract class IndicesHandler(protected val context: CommonBackendConte
 
 internal class ArrayIndicesHandler(context: CommonBackendContext) : IndicesHandler(context) {
```

Comment:
Suggestion: Rename to `CollectionIndicesHandler`

Reply:
Good idea, done.